elements/pfe-content-set/test/pfe-content-set.e2e.spec.ts

```diff
@@ -0,0 +1,20 @@
+import type { LitElement } from 'lit';
+
+import { expect } from '@playwright/test';
+import { test } from '@patternfly/pfe-tools/test/playwright/fixtures.js';
+
+const tagName = 'pfe-content-set';
+
+test.describe(tagName, () => {
```

Comment:
there's a new template for these, check out the latest on pfe-tools branch

Reply:
Gotcha, I just copied over what was in the pfe-content-set component from before. None of these probably work.